src/modules/taskStatus.js

```diff
@@ -0,0 +1,22 @@
+function taskStatus(event) {
+  const list = JSON.parse(localStorage.getItem('Tasks'));
+  const taskChecked = Number(event.currentTarget.parentNode.parentNode.classList[0].slice(5));
+  if (!list[taskChecked].completed) {
+    event.currentTarget.parentNode.parentNode.firstChild.lastChild.style.textDecoration = 'line-through';
+  } else {
+    event.currentTarget.parentNode.parentNode.firstChild.lastChild.style.textDecoration = 'none';
```

Comment:
Please double-check this functionality. It gives an error in all browsers.

![image](https://user-images.githubusercontent.com/26911718/166323825-77cd7be5-a3d2-424f-b9ca-0c2e5176dc88.png)

Reply:
Yeah, this happens because it's looking for a parent Node that still doesn't exist, but is then created so the error doesn't persist.